rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/BaseTypeController.java

```diff
@@ -38,6 +39,9 @@ public BaseTypeController(FieldTypeRepository fieldTypeRepository,
         @ApiResponse(code = 200, message = "All valid base types")
     })
     public List<FieldType> getBaseTypes() {
-        return baseTypes.stream().map(entityToResponseDTOMapper::map).collect(Collectors.toList());
+        if (baseTypesAtomicReference == null) {
```

Comment:
not sure this will work. I think if you check for null, we still could have the original problem.

Reply:
DONE, please, can you review now. thank you